### ISSUE TITLE
Validation not working correctly due to hardcoded digest

### ISSUE THREAD

**Issue by mtamboli** [kabanero-io/kabanero-pipelines#303]
Below validation should have worked for Open Liberty. But it is failing as it seems like `37cdf72ab9589e4b0f1389e1c6acc574a1aa8c9cc2234f6be3fd2e994e1de93f` is hardcoded and it is not part of open-liberty stack but java micro profile.

```
INFO: Enforcing 'stackPolicy' of 'strictDigest'.

INFO: Read project, stack image, docker host and stack name from .appsody-config.yaml
INFO: Git project config in .appsody-config.yaml... 
INFO: VERSION = 0.2.3
INFO: STACK_IMAGE_REGISTRY = docker.io
INFO: PROJECT = kabanerobeta
INFO: STACK_NAME = java-openliberty
INFO: Successfully read project, stack image, docker host and stack name from .appsody-config.yaml

INFO: Validate stack name & project are present, active in the Kabanero CR
INFO: In the cluster...
INFO: STACK_IMAGE = docker.io/kabanerobeta/java-openliberty
INFO: STACK_IMAGE_REGISTRY = docker.io
INFO: PROJECT = kabanerobeta
INFO: STACK_NAME = java-openliberty
INFO: Sucessfully validated stack name & project are present, active in the Kabanero CR

INFO: VERSIONS = 0.2.3
INFO: DIGESTS  = 37cdf72ab9589e4b0f1389e1c6acc574a1aa8c9cc2234f6be3fd2e994e1de93f
INFO: RECONCILED_STACK_IMAGE_REGISTRY_HOST = docker.io
INFO: Cluster stack digest: 37cdf72ab9589e4b0f1389e1c6acc574a1aa8c9cc2234f6be3fd2e994e1de93f
INFO: Project stack version: 0.2.3, Project stack digest: "sha256:8a2050f8f0190a1712d6e033d134c612f234c85efa6a7c6b6b81786b053e4707"

ERROR: A compatible version of the application stack, kabanerobeta/java-openliberty:0.2.3, is not active on this cluster. Please review the active versions of the stack on the cluster (oc get stack java-openliberty -o json) and the stack specification in the .appsody-config.yaml file of the git project
```

**Comment by mtamboli** [comment]
Talked to @smcclem about above problem.

**Comment by mtamboli** [comment]
@smcclem @kvijai82 It seems like this is affecting all of the stacks. Is it possible to get this fixed in a pipeline release? If there is a workaround I can try on my own, please let me know.

**Comment by mtamboli** [comment]
Verified that problem is resolved in pipelines 0.8.0 rc2 release.
